composer.json

```diff
@@ -32,7 +32,7 @@
         "stof/doctrine-extensions-bundle": "^1.2.2",
         "gedmo/doctrine-extensions": "~2.4",
         "symfony-cmf/routing-bundle": "1.2.*",
-        "symfony/symfony": "~2.8.7, !=2.8.10",
+        "symfony/symfony": "^2.8.7, !=2.8.10 || ^3.0",
```

Comment:
we have planned to release 1.5 with symfony ^3.0 but that was not possible because `symfony-cmf` guys are not compatible and we dont want to be dependend on a RC release in 1.5. so we hopefully will be compatible soon in dev-develop and with 1.6